packages/components/inputs/money-input/src/money-input.styles.ts

```diff
@@ -11,10 +11,14 @@ const getCurrencyLabelStyles = () => css`
   border-bottom-left-radius: ${designTokens.borderRadiusForInput};
   border: 1px ${designTokens.borderColorForInputWhenDisabled} solid;
   border-right: 0;
-  padding: 0 ${designTokens.spacing20};
+  padding: ${designTokens.paddingForMoneyInputCurrencyDropdown};
   align-items: center;
   font-size: ${designTokens.fontSizeForInput};
   box-sizing: border-box;
+
+  &:hover {
```

Comment:
Let me pin it here, this comment is not directly related to this line.

I'm not sure if this if OK, but I noticed that when `isReadOnly` is truthy:
the border of `<CurrencyLabel>` is 1px wide
<img width="379" alt="Screenshot 2023-02-06 at 12 22 06" src="https://user-images.githubusercontent.com/49066275/216959147-232a94a5-8c2a-4f24-837c-d51582d44fd2.png">

Reply:
Updated [11712c6](https://github.com/commercetools/ui-kit/pull/2397/commits/11712c63c56868b44f14f3e3f2e0bfd229f7aa4f)